### PR TITLE
add highlights for 2022-10-20

### PR DESCRIPTION
**Pull request by weatherhog**
This Weeks Highlights PR

### CODE REVIEW

**Comment by T-Kukawka** [comment]
Nothing from Phoenix

**Comment by gianfranco-l** [comment]
nothing from honeybadger for this week

File: src/content/changes/highlights/2022-10-20.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: 2022-10-19T10:00:00
```
Comment (marians) [inline src/content/changes/highlights/2022-10-20.md:2]:
```suggestion
date: 2022-10-20T10:00:00
```

File: src/content/changes/highlights/2022-10-20.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+## General
+
+## User interfaces
+
+## Apps
+
```
Comment (marians) [inline src/content/changes/highlights/2022-10-20.md:14]:
```suggestion
## Apps

- [dex-app v1.30.0](https://github.com/giantswarm/dex-app/releases/tag/v1.30.0) provides Dex v2.35.3.
```

**Review by marians — APPROVED** [review]

**Pull request merged**